source/configuration_manual/authentication/ldap_userdb.rst

```diff
@@ -0,0 +1,191 @@
+.. _authentication-ldap_userdb:
+
+==================
+LDAP user database
+==================
+
+Usually your LDAP database also contains the :ref:`userdb
+information <authentication-user_database>`
+If your home directory can be specified with a template and you're using
+only a single :ref:`UID and GID <authentication-user_database>`, you
+should use :ref:`static userdb <authentication-static_user_database>`
+instead to avoid an unnecessary LDAP lookup. You can also use :ref:`prefetch
+userdb <authentication-prefetch_userdb>` to avoid the userdb LDAP lookup.
+
+Userdb lookups are always done using the default DN (``dn`` setting)
```

Comment:
Edit 2: I assume readers know, that it's about LDAP, and that it is a LDAP name. Although I personally would always clarify abbreviations because I read them so frequently I get confused quickly. :astonished: